### ISSUE TITLE
Fatal Error: poll() timeout... tcpdump seems to be having a problem keeping up

### ISSUE THREAD

**Issue by TangMonk** [appneta/tcpreplay#398]
```
$ sudo tcpreplay -v  --intf1=en0 /Users/tangmonk/Downloads/3.pcap
<�'Y°��V
d/�
��V)Evl@@�G�r�E��P�J�	��P ��GET / HTTP/1.1
Host: 114.215.69.217
User-Agent: curl/7.43.0
Accept: */*


Fatal Error:
poll() timeout... tcpdump seems to be having a problem keeping up
Try increasing TCPDUMP_POLL_TIMEOUT
```

this is my pcap file: https://www.dropbox.com/s/gcqdgn4rbwcax62/3.pcap?dl=0

```
sudo tcpreplay -V
tcpreplay version: 4.2.6 (build git:v4.2.6)
Copyright 2013-2017 by Fred Klassen <tcpreplay at appneta dot com> - AppNeta
Copyright 2000-2012 by Aaron Turner <aturner at synfin dot net>
The entire Tcpreplay Suite is licensed under the GPLv3
Cache file supported: 04
Compiled against libdnet: 1.12
Compiled against libpcap: 1.3.0
64 bit packet counters: enabled
Verbose printing via tcpdump: enabled
Packet editing: disabled
Fragroute engine: enabled
Injection method: bpf send()
Not compiled with netmap
```

**Comment by fklassen** [comment]
This is a long-standing bug from before I took over the project. If I understand the history, changes in `tcpdump` brought on this error. I'll investigate for 4.2.7.

**Comment by fklassen** [comment]
[3.pcap.txt](https://github.com/appneta/tcpreplay/files/1032410/3.pcap.txt)

**Comment by sadir** [comment]
Hi @fklassen! Any updates on this? I've just started playing with tcpreplay and I'm stumbling across this error.

**Comment by fklassen** [comment]
I had a quick look, and am concerned about comments from the previous maintainer. It appears that a fix has been attempted, but failed.

I have a backlog of bugs that need lots of testing and am deferring to Sept. We have a code freeze going on right now as we are attempting to get a higher level of stability for current and upcoming Debian/Ubuntu releases. Once stability is achieved, we plan to dig into these tougher issues.

**Comment by sadir** [comment]
Thank you for the quick reply and concise, clear answer! I won't chase any further but do please update as and when you have something to share.

**Comment by fklassen** [comment]
fixed in PR #457

**Comment by sadir** [comment]
@fklassen thank you :)
